blazor/pdfviewer-2/how-to/print-the-sfpdfiewer-inside-the-dialog-component.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Print the SfPdfViewer inside the Dialog in Blazor | Syncfusion
+title: Print the SfPdfViewer inside the Dialog component | Syncfusion
```

Comment:
We can simplify the title in the #single hash and need to check the content PDFViewer is used. Modify it.